Dockerfile

```diff
@@ -12,10 +12,7 @@ ENV SBT_VERSION 0.13.8
 
 # Install Scala
 RUN \
-  cd /root && \
-  curl -o scala-$SCALA_VERSION.tgz http://downloads.typesafe.com/scala/$SCALA_VERSION/scala-$SCALA_VERSION.tgz && \
-  tar -xf scala-$SCALA_VERSION.tgz && \
-  rm scala-$SCALA_VERSION.tgz && \
+  curl -fsL http://downloads.typesafe.com/scala/$SCALA_VERSION/scala-$SCALA_VERSION.tgz |tar xfz - -C /root/ && \
```

Comment:
Please insert a blank btw `|` and `tar`.

Please explain the purpose of the `z` flag and the hyphen before the `-C` flag.